### PR TITLE
Add VersionBot changes to repository.

### PR DESCRIPTION
**Pull request by hedss**
Connects-to: #56
Change-type: patch

### CODE REVIEW

**Review by LucianBuzzo — APPROVED** [review]

**Comment by nghiant2710** [comment]
@hedss can you please update the version here as well https://github.com/resin-io/resin-sdk-python/blob/master/resin/__init__.py#L18?

**Comment by hedss** [comment]
@nghiant2710 Ah, well this raises another interesting point then, which is that the `versionist.conf.js` script needs to be altered so that it updates the version number in the `__init__.py` file to do this. I've simply not got the time to do this atm, so you're probably going to have to do it. :-/

**Comment by nghiant2710** [comment]
@hedss no worries, I will add a commit to update it

**Comment by lurch** [comment]
> Ah, well this raises another interesting point then, which is that the versionist.conf.js script needs to be altered so that it updates the version number in the `__init__.py` file to do this.

Links to https://github.com/resin-io/versionist/issues/84  :wink:

**Comment by lurch** [comment]
> you're probably going to have to do it. :-/

https://github.com/resin-io/versionist/pull/86/files shows an example of how you might do this.

**Comment by hedss** [comment]
@lurch Thanks Andrew!

**Comment by nghiant2710** [comment]
@hedss please merge this one after https://github.com/resin-io/versionist/pull/100 is deployed.

File: versionist.conf.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+};
+
+const isIncrementalCommit = (changeType) => {
+  return Boolean(changeType) && changeType.trim().toLowerCase() !== 'none';
+};
+
+module.exports = {
+  // This setup allows the editing and parsing of footer tags to get version and type information,
+  // as well as ensuring tags of the type 'v<major>.<minor>.<patch>' are used.
+  // It increments in a semver compatible fashion and allows the updating of NPM package info.
```
Comment (lurch) [inline versionist.conf.js:18]: minor nit, but perhaps the `NPM` in the line above should be changed to `Python`? ;-)

**Pull request merged**